integration/dask/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Oof, I missed that we were putting integration testing unrelated to the Arrow binary protocol in integration/. I don't think it makes sense to organize the tests like this. 

https://issues.apache.org/jira/browse/ARROW-3975

Might make sense to put the protocol tests under `format/`

Reply:
I thought integration is the right place for 3rd party integration tests as well (I mean it's still better then `dev`).